core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -105,14 +83,14 @@
    */
   public static final String FIELD_COUNT_PROPERTY_DEFAULT = "10";
 
-  int fieldcount;
+  private int fieldcount;
```

Comment:
For these it would make more sense to add public getters/setters. It would break compatibility for now but would be better long term. I can add the getters/setters and push.